packages/client/hmi-client/src/services/project.ts

```diff
@@ -156,4 +176,34 @@ async function home(): Promise<IProject[] | null> {
 	}
 }
 
-export { create, update, get, remove, getAll, addAsset, deleteAsset, getAssets, home };
+/**
+ * Get the icon associated with an Asset
+ */
+const icons = new Map([
+	[ProjectAssetTypes.DOCUMENTS, 'file'],
+	[ProjectAssetTypes.MODELS, 'share-alt'],
+	[ProjectAssetTypes.DATASETS, 'sliders-v'],
+	[ProjectAssetTypes.SIMULATIONS, 'cog'],
+	[ProjectAssetTypes.SIMULATION_RUNS, 'eye'],
+	[ProjectAssetTypes.CODE, 'code'],
+	['overview', 'inbox']
+]);
+function iconClassname(type: ProjectAssetTypes | string | null): string {
+	if (type && icons.has(type)) {
+		return `pi pi-${icons.get(type)}`;
```

Comment:
Nitpick: How come this is a two step process? As in why not just store `pi pi-foobar` in the map?

Reply:
I could make it in the enum, but it's just repetitive of `pi pi-` so I put it in one place.
Also, we test, because sometimes we can request icons of none exisiting types, so that's why we have a default.